statement.md

```diff
@@ -0,0 +1,84 @@
+# JupyterLab Accessibility Statement
+
+## Edited from the [W3C accessibility statement generator](https://www.w3.org/WAI/planning/statements/generator/#create)
+
+
+## ⚠️ Draft Accessibility Statement for JupyterLab
+
+This is an accessibility statement from <span class="basic-information organization-name">Jupyter accessibility contributors</span>.
```

Comment:
There seems to be a few `<span>` and the such straggling - do they need removing?